app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -42,7 +44,14 @@ export default class SampleViewV2 extends React.Component {
   constructor(props) {
     super(props);
 
-    let localState = this.loadState(localStorage, "SampleViewOptions");
+    this.urlParser = new UrlQueryParser({
+      pipelineVersion: "string",
+      selectedOptions: "object",
```

Comment:
Should `view` be included here? I notice it is picked in `updateHistoryAndPersistOptions`.